docs/excel/custom-functions-json-autogeneration.md

```diff
@@ -320,7 +347,7 @@ A streaming function can indicate an error by calling `setResult()` with an Erro
 
 ### Promise
 
-A function can return a Promise, that provides the value when the promise is resolved. If the promise is rejected, then it will throw an error.
+A custom function can return a promise that provides the value when the promise is resolved. If the promise is rejected, then the custom function will throw an error.
```

Comment:
Worth asking the group as a whole if we consistently use lower case p. I think that is true although I have also seen uppercase in other documentation, so I am just curious!

Reply:
Ok, I'll bring it up at a team meeting! [Term studio](https://termstudio.azurewebsites.net/viewdata.aspx?tid=2254443&langCode=English) presents promise as a lowercase noun (not a proper noun).